connect/models/event.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
```

Comment:
Commit message "Null fields" does not really matches support of events.

Reply:
They have simply been moved from other source files into their own file.